untitled1/src/main/com/aca/homework/week9/expressioncalculator/OperationType.java

```diff
@@ -0,0 +1,45 @@
+package com.aca.homework.week9.expressioncalculator;
+
+public enum OperationType {
+    ADDITION {
+        @Override
+        public double calculate(int number1, int number2) {
+            return number1 + number2;
+        }
+    },
+    SUBTRACTION {
+        @Override
+        public double calculate(int number1, int number2) {
+            return number1 - number2;
+        }
+    },
+    DIVISION {
+        @Override
+        public double calculate(int number1, int number2) {
+            return (double) number1 / number2;
+        }
+    },
+    MULTIPLICATION {
+        @Override
+        public double calculate(int number1, int number2) {
+            return number1 * number2;
+        }
+    };
+
+    public abstract double calculate(int number1, int number2);
+
+    public static OperationType charConvertToOperationType(char type) {
+        switch (type) {
+            case '+':
```

Comment:
let's create a constructor field, getter for that field, and by using a for loop find the enum value for the provided char type